packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -627,4 +743,121 @@ mod tests {
         remove_base_id(&mut db, property_type.inner().id()).await?;
         Ok(())
     }
+
+    fn book_entity_type_v1() -> EntityType {
+        serde_json::from_value(serde_json::json!({
+            "kind": "entityType",
+            "$id": "https://blockprotocol.org/types/@alice/entity-type/book/v/1",
+            "title": "Book",
+            "type": "object",
+            "properties": {
+                "https://blockprotocol.org/types/@alice/property-type/name": {
+                    "$ref": "https://blockprotocol.org/types/@alice/property-type/name"
+                },
+                "https://blockprotocol.org/types/@alice/property-type/blurb": {
+                    "$ref": "https://blockprotocol.org/types/@alice/property-type/blurb"
+                },
+                "https://blockprotocol.org/types/@alice/property-type/published-on": {
+                    "$ref": "https://blockprotocol.org/types/@alice/property-type/published-on"
+                }
+            },
+            "required": [
+                "https://blockprotocol.org/types/@alice/property-type/name"
+            ],
+        }))
+        .expect("Invalid entity type")
+    }
+
+    fn book_entity_type_v2() -> EntityType {
+        // TODO: Update version number
+        serde_json::from_value(serde_json::json!({
+            "kind": "entityType",
+            "$id": "https://blockprotocol.org/types/@alice/entity-type/book/v/1",
+            "title": "Book",
+            "type": "object",
+            "properties": {
+                "https://blockprotocol.org/types/@alice/property-type/name": {
+                    "$ref": "https://blockprotocol.org/types/@alice/property-type/name"
+                },
+                "https://blockprotocol.org/types/@alice/property-type/blurb": {
+                    "$ref": "https://blockprotocol.org/types/@alice/property-type/blurb"
+                },
+                "https://blockprotocol.org/types/@alice/property-type/published-on": {
+                    "$ref": "https://blockprotocol.org/types/@alice/property-type/published-on"
+                }
+            },
+            "required": [
+                "https://blockprotocol.org/types/@alice/property-type/name"
+            ],
+            "links": {
+                "https://blockprotocol.org/types/@alice/property-type/written-by": {}
+            },
+            "requiredLinks": [
+                "https://blockprotocol.org/types/@alice/property-type/written-by"
+            ],
+        }))
+        .expect("Invalid entity type")
+    }
+
+    #[tokio::test]
+    #[cfg_attr(miri, ignore = "miri can't run in async context")]
+    async fn create_entity_type() -> Result<(), DatastoreError> {
+        let mut db = PostgresDatabase::new(&DB_INFO).await?;
+        let account_id = create_account_id(&db.pool).await?;
+
+        let entity_type = db
```

Comment:
I added the clean-up comment. Where you want me to put the other comment? The test is literally called like this.

Reply:
Maybe the `// Clean up` is good enough, I was thinking we could do the `// Check ...` just above `let entity_type`, because the rest of the stuff above is setup and it's not what we're trying to test

Reply:
This would result in 
```rust
/// Check we can create an entity type
let entity_type = db.create_entity_type(...)?;
```

I honestly don't see any value in it here.